engines/cruise/cruise_main.cpp

```diff
@@ -1431,47 +1434,47 @@ int CruiseEngine::processInput() {
 			else if (pausedButtonDown)
```

Comment:
I think this mouse status check could be replaced if endPause was set to true on mouse up instead of mouse down

Reply:
Done, you were right